doc/tutorials/core/random_generator_and_text/random_generator_and_text.markdown

```diff
@@ -95,7 +95,7 @@ Explanation
 
        line( image, pt1, pt2, randomColor(rng), rng.uniform(1, 10), 8 );
        imshow( window_name, image );
-       if( waitKey( DELAY ) >= 0 )
+       if( (waitKey( DELAY ) & 0xFF) >= 0 )
```

Comment:
Well yeah it does, but ofcourse, then they could simply have used waitKey() right? maybe I should dig into the actual sample to get an understanding of that DELAY parameter